EIPS/eip-4962.md

```diff
@@ -0,0 +1,96 @@
+---
+eip: 4962
+title: DEST Operation
+description: Add the DEST operation to the EVM
+author: Danno Ferrin (@shemnon)
+discussions-to: https://twitter.com/shemnon/status/1509725635576725506
+status: Draft 
+type: standards track
+category : Core
+created: 2022-04-01
+requires : 3670, 3855
+---
+
+## Abstract
+
+Add `DEST` operation to the EVM, allowing control flow code to say where execution comes from.
+
+## Motivation
+
+The EVM has long had a parallel to the controversial GOTO operation ("Edgar Dijksta: Go To Statement
+Considered Harmful", *Communications of the ACM.* 11 (3): 147–148), but there is a lesser known and
+equally powerful operation that has not been implemented: COMEFROM (EIP link policies prevent a
+relevant external link). By implementing the `DEST` operation that implements the COMEFROM semantics
```

Comment:
```suggestion
equally powerful operation that has not been implemented: `COMEFROM`. By implementing the `DEST` operation that implements the COMEFROM semantics
```

Process complaints should be directed to the editors out-of-band, and not included in EIPs themselves.